Makefile

```diff
@@ -247,7 +231,6 @@ check3D_64: all
 	bin/cufinufft3d1_test 4 15 15 15 2048 1e-3
 	bin/cufinufft3d2_test 1 16 16 16 4096 1e-3
 	bin/cufinufft3d1_test 1 128 128 128
-	bin/cufinufft3d1_test 2 16 16 16
```

Comment:
Why gone?

Reply:
3D method 2 has the smem limit.  There is a comment a few lines above to that effect.

It was actually erroneous before, but it wasn't picked up until I started returning `ier`...

Reply:
Ah right. Forgot about that.